components/mpas-seaice/docs/user-guide/index.md

```diff
@@ -0,0 +1,64 @@
+| ``itd``             | ice thickness distribution               |
+| ``floesize``        | floe size distribution                   |
+| ``ridging``         | mechanical redistribution                |
+| ``atmosphere``      | atmospheric boundary layer and coupling  |
+| ``ocean``           | oceanic boundary layer and coupling      |
+| ``diagnostics``     | diagnostic output                        |
+| ``prescribed_ice``  | for testing atmosphere simulations       |
+
+## Icepack
+
+The Icepack software has replaced the original ``colpkg`` column physics code in MPAS-seaice. The ``column_package`` option is still available but is no longer being supported in MPAS-seaice.
+
+Full documentation for E3SM's version of Icepack can be found in [E3SM's Icepack readthedocs](<https://e3sm-icepack.readthedocs.io/en/latest>).  The most up-to-date documentation from the CICE Consortium's main Icepack repository is [here](<https://cice-consortium-icepack.readthedocs.io/en/main>).
+
+The MPAS-seaice driver for Icepack is
+
+``E3SM/components/mpas-seaice/src/shared/mpas_seaice_icepack.f``
+
+and the mapping between the names of Icepack's namelist options and those in MPAS-seaice can be found in subroutine ``init_icepack_package_configs`` (see the argument list for ``call subroutine icepack_init_parameters`` and comments at the end of ``init_icepack_package_configs``.
+
+## Configuring Model Input and Output
+
+The reading and writing of model fields in MPAS is handled by user-configurable streams. A stream represents a fixed set of model fields, together with dimensions and attributes, that are all written or read together to or from the same file or set of files. Each MPAS model core may define its own set of default streams that it typically uses for reading initial conditions, for writing and reading restart fields, and for writing additional model history fields. Besides these default streams, users may define new streams to, e.g., write certain diagnostic fields at a higher temporal frequency than the usual model history fields.
+
+Streams are defined in XML configuration files that are created at build time for each model core. The name of this XML file is simply ‘streams.’ suffixed with the name of the core. For example, the streams for the sw (shallow-water) core are defined in a file named ‘streams.sw’. An XML stream file may further reference other text files that contain lists of the model fields that are read or written in each of the streams defined in the XML stream file.
+
+Changes to the XML stream configuration file will take effect the next time an MPAS core is run; there is no need to re-compile after making modifications to the XML files. As described in the next section, it is therefore possible, e.g., to change the interval at which a stream is written, the template for the filenames associated with a stream, or the set of fields that are written to a stream, without the need to re-compile any code.
```

Comment:
Suggest replacing this with the following:

The stream file can be modified by copying the `streams.seaice` file (generated during `./case.setup`) from the run directory to the case directory under `SourceMods/src.mpassi/`, and making changes to the stream file there. Changes to the stream file will take effect on the next case submission.

Reply:
I updated the text with all of @darincomeau's changes.